packages/app/src/cli/services/dev/ui/components/Dev.tsx

```diff
@@ -206,6 +214,12 @@ const Dev: FunctionComponent<DevProps> = ({
         >
           {canUseShortcuts ? (
             <Box flexDirection="column">
+              {isEditionWeek ? (
+                <Text>
+                  {figures.pointerSmall} Press <Text bold>e</Text> {figures.lineVertical} check out {season} Edition
+                  {` ${year}`}, live NOW with 100+ product announcements!
```

Comment:
We will show this message for a week, so `live NOW` won't be very accurate... Why not simply showing `check out Winter Edition 2024 with 100+ product announcements!`?

Reply:
Well, I don't think `live NOW` implies it just went live, but rather that at the current moment it's live.

Probably not as exciting after a few days, but I don't think anyone will complain... also keep in mind people might not even see it for a few days if they don't upgrade CLI. So it's only for people who are actually using the latest (creating new apps or upgrading).